build.gradle

```diff
@@ -64,6 +64,9 @@ repositories {
 dependencies {
 	api platform('com.exactpro.th2:bom:4.0.2')
 
+	compile "org.slf4j:slf4j-api"
+	compile "org.slf4j:slf4j-log4j12"
+
 	compile "io.grpc:grpc-protobuf"
 	compile "io.grpc:grpc-stub"
 	compile "io.grpc:grpc-core"
```

Comment:
Why do we use compile here? It should be `implementation`. Also, I think we don't need to specify them explicitly because we will have them as transitive dependencies from common lib